.gitignore

```
@@ -23,7 +23,7 @@ awscpu
 # ignore GoLand metafiles directory
 .idea/
 
-*logs/
+logs/
```

nit maybe

```suggestion
*logs/
!*customlogs/
```

so that other logs/ directories are ignored still?